### DISCUSSION TITLE
Dual F/C display

### DISCUSSION THREAD

**Discussion by cliffr39** [PranshulGG/WeatherMaster#182] [Ideas]
One of my favorite features of Weawow weather is that it can display both F/C next to each other. Would you please implement that. I use both measuring systems, but my wife gets confused when she looks at my phone (lol).

**Comment by PranshulGG** [comment]
> One of my favorite features of Weawow weather is that it can display both F/C next to each other. Would you please implement that. I use both measuring systems, but my wife gets confused when she looks at my phone (lol).

Adding this might take a while since there’s currently no space for it, and it could mess up the UI. I’ll see if I can fit it in without disrupting the layout, though!

**Comment by cliffr39** [comment]
Sorry don't misunderstand. I don't mean to re-write all areas of the app, just the top "current" like this below. Ignore how crude it was done, just a thought and certainly don't expect you to implement.

![com_example_weathermaster_MainActivity](https://github.com/user-attachments/assets/40d0b2b7-338b-4d2d-9ab1-51ab9ce03ed1)
